src/precision.cpp

```diff
@@ -120,6 +123,7 @@ Precision fromKfsString(const std::string& s) {
         {"UINT16", Precision::U16},
         {"UINT8", Precision::U8},
         {"BOOL", Precision::BOOL},
+        {"BYTES", Precision::STRING},
```

Comment:
This effectively stops us from using bytes for raw image/audio? Right now OV does not support such formats, but there are models in TF that accepted image as input.

Shouldn't we use KFS parameters additionally to point out what type of data is actually in RESPONSE/REQUEST?

Reply:
ov::Tensor <-> PyTensor conversion will stop being "undefined" for BYTES. therefore we end up with memcpy of strings which is undefined behavior

Reply:
TODO: Move fromKfsString to converter entirely. It is not related to core mappings. Remove the mapping from

Reply:
Done